server/stycoon/ships.go

```diff
@@ -0,0 +1,28 @@
+package stycoon
+
+import (
+	"fmt"
+	"strconv"
+)
+
+func (game *Game) SetShipClasses() error {
+	shipClasses := make(map[string]StaticDataShipClassesValue)
+	rows, err := game.db.Query("select * from d_class")
```

Comment:
perhaps it would be better to explicitly list the columns here?

Reply:
yeah I thought about it, I will explicitly list it for all selects

Reply:
done